app/components/ideas.hbs

```diff
@@ -1,8 +1,8 @@
-<div class="card-columns">
-  <Ideas::Filter @ideas={{@ideas}} @query={{@query}} as |results| >
-    {{#each results as |idea|}}
-    <Idea @idea={{idea}} />
-    {{/each}}
-  </Ideas::Filter>
+<div class="card__wrap--outer">
+    <Ideas::Filter @ideas={{@ideas}} @query={{@query}} as |results| >
+      {{#each results as |idea|}}
+      <Idea @idea={{idea}} />
+      {{/each}}
+    </Ideas::Filter>
```

Comment:
Fix indent to 2 spaces

Reply:
You marked this `resolve`, but I don't see fix of that, maybe you not pushed changes? :)